src/atoms/label/Label.js

```diff
@@ -2,7 +2,7 @@
 export default {
   props: {
     /**
-     * Label tag
+     * To use another tag instead of 'label'
      */
     tag: {
```

Comment:
IMO custom tag is unnecessary, WDYT is there any use case for custom tag?

Reply:
not really, I'll change component to label